code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1135,6 +1135,50 @@ datum
 				..()
 				return
 
+		harmful/hydrogen_sulfide
+			name = "hydrogen sulfide"
+			id = "hydrogen_sulfide"
+			description = "A dangerous asphyxiant substance that inhibits cellular respiration."
+			reagent_state = SOLID
+			fluid_r = 255
+			fluid_g = 255
+			fluid_b = 255
+			transparency = 0
+			depletion_rate = 0.2
+			penetrates_skin = 1
```

Comment:
slow-depletion skinpen poison. Just what we want to keep adding 🙄

Reply:
It takes 12 units to reach crit and it remains dormant for the first 24 seconds, i would say double the depletion of cyanide for a poison who's lack of direct effect really makes stacking counterproductive fair. If you're worried this will somehow make skinpen mixes stronger, i may remind you that any big stack would have had you close to crit by the time this does anything.

Reply:
Another thing i would like to point out is oxygen deprivation takes much longer to actually kill you, so even though it takes 12 units to take you into critical condition, you can stay alive for up to 30 units.

Reply:
This was changed with a damage buff at the late stages, now it will take 22 units in you to kill you, which is about 3.7 minutes, as opposed to cyanide's about 2.8 minutes(8 to 9 units of cyanide is generally what is depleted before it kills you).